physx.js

```diff
@@ -612,6 +612,74 @@ const physxWorker = (() => {
       } : null;
     }
   };
+  w.raycastPhysicsArray = (physics, p, q, n) => {
+    if (physics && (n > 0 && n < 1000)) {

```

Comment:
If we are going to guard then we should at least throw an error if it's violated.

Reply:
I have added throw new Error('raycastPhysicsArray error'); to solve this issue